src/common/depthmap_toolkit/depthmap.py

```diff
@@ -61,16 +62,19 @@ def process(plt, dir_path, depth, rgb):
     utils.parse_data('data')
 
     # read rgb data
+    global current_rgb
```

Comment:
global variables should be uppercase
```suggestion
    global CURRENT_RGB
```

See more occurrences below